metadata.json

```diff
@@ -31,21 +31,21 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 5.5.8 < 7.0.0"
+      "version_requirement": ">= 6.1.0 < 8.0.0"
     }
   ],
   "dependencies": [
     {
       "name": "puppetlabs/stdlib",
-      "version_requirement": ">= 4.13.1 < 7.0.0"
+      "version_requirement": ">= 4.13.1 < 8.0.0"
     },
     {
       "name": "puppet/python",
-      "version_requirement": ">= 4.1.0 < 6.0.0"
+      "version_requirement": ">= 4.1.0 < 7.0.0"
     },
     {
       "name": "puppetlabs/vcsrepo",
-      "version_requirement": ">= 1.3.1 < 4.0.0"
+      "version_requirement": ">= 1.3.1 < 5.0.0"
```

Comment:
The lower bound should also change to `4.0.0` as they dropped puppet 5, same with stdlib which should change to `7.0.0` and python should be `6.0.0`